specification/ingest/_types/Database.ts

```diff
@@ -19,15 +19,21 @@
 
 import { Id, Name } from '@_types/common'
 
-export class DatabaseConfiguration {
+specification / ingest / _types / Database.ts
```

Comment:
I don't know why this does not break CI, but let's remove it anyway:

```suggestion
```

Reply:
Gah, yeah, that was just sloppiness on my part. 91e9c648f2ffdb5184e1ad0f35d0d1af4c684519